s3bench.py

```diff
@@ -128,6 +132,12 @@ def time_operation(self, method, object_name, bin_data):
 
         return diff
 
+    def evaluate_latency(self, op_duration):
```

Comment:
1. we should initialize the self.max_latency in the class vars to match a ternary if that looks like the following:
 ```self.max_latency = args.max_latency if args.max_latency else 0```
Then we could just use return 0/1 here without using the actual flag

Why? 
1. if the customer didn't use the `--max-latency` flag, the `float(self.max_latency)` check will fail with `NonType` object 
2. We should keep the method shorter

```suggestion 
def evaluate_latency(self, op_duration):
    if op_duration > float(self.max_latency):
        return 1
    return 0
```

Reply:
seems reasonable